pkg/cmd/pulumi/packagecmd/package_publish_sdk.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2016-2024, Pulumi Corporation.
```

Comment:
This file was just moved, wasn't created in this PR.

Reply:
Using `git mv` instead of trusting `git` to figure this out would help here.

Reply:
That's actually what I did and it shows up properly if you only look at the first commit. I think the problem here is the diffing algorithm GitHub uses

Reply:
Yeah, `git mv` doesn't actually do anything different than just `mv` + `git add`.  There's no internal tracking in git of moved files.

Reply:
I solved it by splitting the PRs into multiple